app/scripts/modules/amazon/src/pipeline/stages/deployCloudFormation/evaluateCloudFormationChangeSetExecutionDetails.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+import { IExecutionDetailsSectionProps } from 'core/pipeline';
+import { IStage } from 'core/domain';
+import { ExecutionDetailsSection } from '@spinnaker/core';
+import {
+  EvaluateCloudFormationChangeSetExecutionApproval,
+  IEvaluateCloudFormationChangeSetExecutionApprovalState,
+} from './evaluateCloudFormationChangeSetExecutionApproval';
+
+export interface IEvaluateCloudFormationChangeSetExecutionDetailsState {
+  parentDeployStage: IStage;
+}
+
+export class EvaluateCloudFormationChangeSetExecutionDetails extends React.Component<
+  IExecutionDetailsSectionProps,
+  IEvaluateCloudFormationChangeSetExecutionApprovalState
+> {
+  public static title = 'Change Set Execution';
```

Comment:
This isn't being used anywhere that I can tell?

Reply:
That is the title of the details tab, should this be defined anywhere else?

![Screenshot 2019-12-19 at 15 25 51](https://user-images.githubusercontent.com/1881230/71181308-2a553780-2274-11ea-85c2-41f02087c114.png)